packages/colibri/src/project_manager/project_manager.ts

```diff
@@ -501,8 +505,21 @@ export class Project_manager {
 
         return await this.tools_manager.get_test_list(this.get_project_definition(n_config_manager));
     }
-}
-
 
+    public getBuildSteps(): t_taskRep[] {
+        return [];
+    }
 
+    // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
+    public runTask(_taskType: e_quartusTaskType, _callback: (result: p_result) => void): ChildProcess {
```

Comment:
there should be a generic tasktype so we can use it here, instead of the quartus one

Reply:
done